src/EnergyPlus/HVACDXHeatPumpSystem.cc

```diff
@@ -1214,18 +1201,14 @@ namespace HVACDXHeatPumpSystem {
         // PURPOSE OF THIS SUBROUTINE:
         // Get Outlet node number
 
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int NodeNum;
-        int DXHeatSysNum;
```

Comment:
[src/EnergyPlus/HVACDXHeatPumpSystem.cc:1219]:(style),[variableScope],The scope of the variable 'DXHeatSysNum' can be reduced.